tests/components/sonos/conftest.py

```diff
@@ -162,11 +162,16 @@ class MockSoCo(MagicMock):
     surround_level = 3
     music_surround_level = 4
     soundbar_audio_input_format = "Dolby 5.1"
+    partymode_called = False
 
     @property
     def visible_zones(self):
         """Return visible zones and allow property to be overridden by device classes."""
         return {self}
+        
+    def partymode(self):
+        """ Fake call partymode services """
```

Comment:
While this looks like it will work, we shouldn't have to do anything.  The mock should absorb the service call and then report whether it is called via sock_mock.partymode.call_count.

There are other example is test_media_player.py

```
    assert soco_mock.clear_queue.call_count == 1
    assert soco_mock.add_to_queue.call_count == 1
```